pkg/parser/containerd.toml.gtpl

```diff
@@ -0,0 +1,118 @@
+version = 2
```

Comment:
what about the containerd config with no GPU?

Reply:
since the plan is to deprecate the skip-gpu-driver-install tag soon now that the api field is available, i don't think it is worth adding the same hacky way of doing it now since the api field will mostly likely be GA by the time RP is using self-contained anyway. But i can add it back if you think its best to keep consistency for now.

Reply:
Oh i realize i need to update the `if` condition in the config though to work the skip gpu driver if we're going to get rid of the other config though. Just updated.